src/main/java/com/synopsys/integration/alert/channel/email/EmailMessagingService.java

```diff
@@ -54,6 +54,8 @@
 import freemarker.template.TemplateException;
 
 public class EmailMessagingService {
+    private static final String SIMPLE_EMAIL_ADDRESS_PATTERN = ".{1,64}@.+";
```

Comment:
There should be some already existing validators we can use to verify Email addresses.

Reply:
`  try {
                final InternetAddress emailAddr = new InternetAddress(testEmailAddress);
                emailAddr.validate();
            } catch (final AddressException ex) {
                throw new AlertException(String.format("%s is not a valid email address. %s", testEmailAddress, ex.getMessage()));
            }`